newui/docs/_config.yml

```diff
@@ -0,0 +1,218 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your entire site, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# `jekyll serve`. If you change this file, please restart the server process.
+
+#
+hudi_style_skin          : "hudi"
+
+version : &version "0.5.1-SNAPSHOT"
+
+previous_docs:
+  latest: /docs/quick-start-guide.html
+  0.5.0-incubating: /versions/0.5.0-incubating/docs/quick-start-guide.html
+
+
+# Site Settings
+locale                   : "en-US"
+title                    : "Apache Hudi"
+title_separator          : "-"
+subtitle                 : *version
+description              : "Apache Hudi Stands for Hadoop Upserts and Incrementals to manage the storage of large analytical datasets on HDFS."
+url                      : https://hudi.apache.org # the base hostname & protocol for your site e.g. "https://mmistakes.github.io"
+repository               : "apache/incubator-hudi"
+teaser                   : "/assets/images/500x300.png" # path of fallback teaser image, e.g. "/assets/images/500x300.png"
+logo                     : "/assets/images/hudi.png" # path of logo image to display in the masthead, e.g. "/assets/images/88x88.png"
+masthead_title           : # overrides the website title displayed in the masthead, use " " for no title
+host                     : 0.0.0.0
+site_url                 : https://hudi.apache.org
+
+# Site QuickLinks
+author:
+  name             : "Quick Links"
+  bio              : "Apache Hudi stands for *Hadoop* *Upserts* *Deletes* and *Incrementals*."
```

Comment:
Since this is a large PR, it will be difficult to review line by line... Even within this file, what Hudi stands for is written differently :). can you take a closer pass and match all content verbatim from the current site>?

Reply:
Done.